src/lib/component/partial/ToolbarSelect/Button.js

```diff
@@ -1,7 +1,7 @@
 import React from "/vendor/react";
 import PropTypes from "prop-types";
 
-import { button as BaseButton } from "/vendor/@material-ui/core";
+import { Button as BaseButton } from "/vendor/@material-ui/core";
```

Comment:
Capitalization is needed because that how the export from `"/vendor/@material-ui/core"` is defined.